.circleci/config.yml

```diff
@@ -39,8 +39,15 @@ common_test_steps: &common_test_steps
           # This should cache the npm cache instead of node_modules, which is needed because
           # npm ci actually removes node_modules before installing to guarantee a clean slate.
           - ~/.npm
-    - run: npm run test:ci
+    - run:
+        command: npm run test:ci
+        environment:
+          JEST_JUNIT_OUTPUT: "reports/junit/js-test-results.xml"
```

Comment:
![image](https://user-images.githubusercontent.com/841294/46105870-d332f500-c1df-11e8-8423-23d386b251b8.png)

Nice to have these reports broken out separately in CircleCI, rather than having to dig through the test output!